drivers/can/Kconfig

```diff
@@ -36,6 +36,15 @@ config CAN_WORKQ_FRAMES_BUF_CNT
 	help
 	  Number of frames in the buffer of a zcan_work.
 
+config CAN_RX_TIMESTAMP
+	bool "Enable receiving timestamps"
+	depends on CAN_STM32
+	help
+	  This options enable a timestamp value of the CAN free running timer.
+	  The value is incremented every bit time and starts when the controller
+	  is initialized.
+	  The maximum value is hardware dependent (stm32: 16bit, mcux: 16bit).
```

Comment:
Is this always 16 bit? If not, should we just use int in zcan_frame struct?

Reply:
The timestamp is usually there for TTCAN which always uses 16-bit. The CANcan controllers we have, only support 16-bit or no timestamp (MCP2515). If we introduce an int, we would need to provide the max-value. Providing the max-value would need a new API because external controllers could differ from internal.

Reply:
Updated the comment.